packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -1195,9 +1195,9 @@ export function createLodestarMetrics(
         help: "Histogram of cloned count per state every time state.clone() is called",
         buckets: [1, 2, 5, 10, 50, 250],
       }),
-      statePersistDuration: register.histogram({
-        name: "lodestar_cp_state_cache_state_persist_seconds",
-        help: "Histogram of time to persist state to db",
+      stateSszDuration: register.histogram({
```

Comment:
nit: would prefer serialize here, using ssz as a abbreviation for serialize might be confusing